From nobody Mon Sep 17 00:00:00 2001
Subject: [PATCH] fix alpha breakage
From: Ivan Kokshaysky <ink@jurassic.park.msu.ru>
Date: 1130634943 -0700

barrier.h uses barrier() in non-SMP case.  And doesn't include compiler.h.

Cc: Al Viro <viro@ftp.linux.org.uk>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Linus Torvalds <torvalds@osdl.org>
Signed-off-by: Chris Wright <chrisw@osdl.org>
---

 include/asm-alpha/barrier.h |    2 ++
 1 files changed, 2 insertions(+)

Index: linux-2.6.14.y/include/asm-alpha/barrier.h
===================================================================
--- linux-2.6.14.y.orig/include/asm-alpha/barrier.h
+++ linux-2.6.14.y/include/asm-alpha/barrier.h
@@ -1,6 +1,8 @@
 #ifndef __BARRIER_H
 #define __BARRIER_H
 
+#include <asm/compiler.h>
+
 #define mb() \
 __asm__ __volatile__("mb": : :"memory")
 
